source/_integrations/lamarzocco.markdown

```diff
@@ -47,16 +47,10 @@ By default, this integration will query your machine every 30 seconds for an upd
 {% configuration_basic %}
 Username:
   description: "Your username you use to log into the La Marzocco app."
-  required: true
-  type: string
```

Comment:
These changes actually don't belong in this PR?

Reply:
next is just behind current atm, so this needs a rebase